code-challenges.test.js

```diff
@@ -50,5 +82,22 @@ var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
 
+describe("scream", () => {
+    var testArray1 = [3, 7, 10, 5, 4, 3, 3]
+    var testArray2 = [7, 8, 2, 3, 1, 5, 4]
+
+    it("take in two array arguments", () => {
+        expect(scream(testArray1)).toEqua[3, 7, 10, 5, 4, 8, 2, 1];
+    })
+})
```

Comment:
I think there are a couple issues with your test. As you noted in your `it` statement, your function is going to take two arrays as arguments. You have two test array variables but only one is being passed to your test. It also looks like there is a small syntax error on your `.toEqual`
`expect(scream(testArray1, testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])`